### PR TITLE
Happy Path Registrastion and Bug Fixes

### PR DESCRIPTION
**Pull request by filcap**
- Fix issues in Eternal Storage with mismatch keys and types in various contracts
- Add getter functions for code used by Registration Tests
- Happy path for registration tests complete

### CODE REVIEW

File: contracts/DeviceManager.sol [hunk 1]
```diff
@@ -77,6 +77,54 @@ contract DeviceManager is Migratable, Pausable, TokenDestructible {
...
+    ///
+    /// STORAGE GETTERS
+    ///
+    /// @notice checks if an account is a manufacturer
+    function isManufacturer(address _account) public view returns(bool) {
+        return atonomiStorage.getBool(
+            keccak256(
+                "network",
+                _account,
+            "isManufacturer")
```
Comment (mraybman-zz) [inline contracts/DeviceManager.sol]: carriage return

File: contracts/DeviceManager.sol [hunk 2]
```diff
@@ -77,6 +77,54 @@ contract DeviceManager is Migratable, Pausable, TokenDestructible {
...
+        );
+    }
+
+    /// @notice get accounts manufacturer id
+    function networkMemberId(address _account) public view returns(bytes32) {
+        return atonomiStorage.getBytes32(
+            keccak256(
+                "network",
+                _account,
+            "memberId")
```
Comment (mraybman-zz) [inline contracts/DeviceManager.sol]: carriage return

File: contracts/DeviceManager.sol (hunk 1 above)
Comment (filcap) [reply contracts/DeviceManager.sol]: fixed

File: contracts/ReputationManager.sol [hunk 3]
```diff
@@ -74,6 +74,18 @@ contract ReputationManager is Migratable, Ownable, Pausable {
         _;
     }
 
+    ///
+    /// STORAGE GETTERS
+    ///
+    /// @notice get default reputation for manufucturer
+    function defaultManufacturerReputations(bytes32 _memberId) public view returns(bytes32) {
```
Comment (mraybman-zz) [inline contracts/ReputationManager.sol]: make reputation singular (requires changing references)

**Review by mraybman-zz — APPROVED** [review]

File: contracts/ReputationManager.sol (hunk 3 above)
Comment (filcap) [reply contracts/ReputationManager.sol]: fixed

File: contracts/DeviceManager.sol (hunk 2 above)
Comment (filcap) [reply contracts/DeviceManager.sol]: fixed

**Pull request merged**